src/css/popup-new.css

```diff
@@ -34,6 +35,12 @@
   padding: var(--spacingMd);
 }
 
+/* Magic Number: Custom max-height for masks panel */
```

Comment:
This doesn't tell me anything about where the number came from :sweat_smile: wy not e.g. 400px?

Reply:
Heh. Revised comment to refactor to make it not an absolute number. It's going to break in future iterations. Leaving as a placeholder for now.

Reply:
Revised in 4021b7f61fa14cabcfea18a7169a3c7ffcc1b4ab